data/Debian.yaml

```diff
@@ -9,3 +9,5 @@ rundeck::repo_config:
     key:
       name: rundeck
```

Comment:
This should fix the repo signature validation problem (should be a separate PR, but I noticed it here):

```suggestion
      name: rundeck.asc
```

Reply:
Alright i will add some tests for the new code and create a separate PR for repo thing